### DISCUSSION TITLE
文件类型打开后是音乐方式

### DISCUSSION THREAD

**Discussion by lianyuxue1020** [AlistGo/alist#857] [Q&A]
请问我通过源码运行时以下文件类型的txt,htm,html,xml,java,properties,sql,js,md,json,conf,ini,vue,php,py,bat,gitignore,yml,go,sh,c,cpp,h,hpp,tsx,vtt,srt,ass在界面预览时以音乐方式打开
![image](https://user-images.githubusercontent.com/63288263/160781513-331ae8dc-78ff-4343-bdbe-36ed2ad14744.png)

**Comment by xhofe** [comment]
v2分支的源码有些小bug 已在dev分支修复。

**Reply by lianyuxue1020** [reply]
我在15点时已经切换到dev分支后仍然存在该问题

**Reply by xhofe** [reply]
你可以看看后台的audio types设置

**Reply by lianyuxue1020** [reply]
![image](https://user-images.githubusercontent.com/63288263/160783225-e459a577-d599-4d5e-9ffb-73d15b9d1a6a.png)
我down下后没有做其他修改，这个需要修改其他内容嘛？

**Reply by xhofe** [reply]
那我不知道了 看不出来你这是哪个提交之后的，我本地dev最新的代码时正常的。

**Reply by xhofe** [reply]
如果你这个版本没有错的 那么说明你这还是v2分支的代码且是https://github.com/Xhofe/alist/commit/466ec27ffe17c25bbb7253b81b82307e8835bc7b 这个提交的

**Reply by lianyuxue1020** [reply]
是这样的。已经解决啦，感谢🙏
